topology/consistency_level.go

```diff
@@ -307,3 +307,68 @@ const (
 	majority         = "majority"
 	unstrictMajority = "unstrict_majority"
 )
+
+// WriteConsistencyAchieved returns a bool indicating whether or not we've received enough
```

Comment:
+1 for having these available from the `topology` package.

Reply:
Yeah good call on this.